dash/testing/plugin.py

```diff
@@ -189,10 +189,18 @@ def dashjl(request, dashjl_server, tmpdir):
 
 @pytest.fixture
 def diskcache_manager():
-    from dash.long_callback import (  # pylint: disable=import-outside-toplevel
-        DiskcacheLongCallbackManager,
-    )
-    import diskcache  # pylint: disable=import-outside-toplevel
+    try:
+        from dash.long_callback import (  # pylint: disable=import-outside-toplevel
+            DiskcacheLongCallbackManager,
+        )
+        import diskcache  # pylint: disable=import-outside-toplevel
+    except ImportError as missing_imports:
+        raise ImportError(
+            """\
+DiskcacheLongCallbackManager requires extra dependencies which can be installed doing
+
+$ pip install "dash[diskcache-manager]"\n"""
+        ) from missing_imports
```

Comment:
🌴  Perhaps make these imports a `@staticmethod` of `DiskcacheLongCallbackManager` so we don't need to repeat this, it's just something like:
```py
    from dash.long_callback import (  # pylint: disable=import-outside-toplevel
        DiskcacheLongCallbackManager,
    )
    diskcache = DiskcacheLongCallbackManager.imports()[0]
```

Reply:
I solved the duplication in a slightly different way: If `cache` is not given to `DiskcacheLongCallbackManager` it defaults to initiate a `diskcache.Cache()` instance with default values (the same as the `pytest` fixture uses). This way the `pytest` fixture will naturally fail with the understandable error message provided centrally by `DiskcacheLongCallbackManager.__init__()` if dependencies are not installed.

Let me know @alexcjohnson if this way of doing it is in conflict some other existing/planned usage patterns with `DiskcacheLongCallbackManager`.

Reply:
@jonmmease any reason not to do it this way? It seems like `diskcache` generally works fine without an explicit directory - which simplifies things for users, unless there's a common case where it fails.

Reply:
Yeah, that sounds fine